src/frontend/pages/blogs/BlogPage.js

```diff
@@ -0,0 +1,206 @@
+import React, { useCallback } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components';
+import UnstyledButton from '../../components/Button';
+import PreviewTable from '../../components/PreviewTable';
+import TableCell from '@material-ui/core/TableCell';
+import useBlogInfo from '../../hooks/blogs'
+import { useHistory } from 'react-router-dom';
+import closeIcon from './assets/closeIcon.svg';
+
+const Button = styled(UnstyledButton)``;
+
+const Container = styled.div`
+  margin: ${(props) => props.theme.pageMargin};
+`;
+
+const CurrentGooseHeader = styled.p`
+  font: ${({ theme }) => theme.fonts.medium24};
+`;
+
+const ButtonContainer = styled.div`
+  ${Button} {
+    margin-right: 28px;
+    box-shadow: ${({ theme }) => theme.shadows.shadow1};
+  }
+`;
+
+const TextBold = styled.p`
+  font: ${({ theme }) => theme.fonts.bold18};
+`;
+
+const SummaryBtn = styled.button`
+  display: flex;
```

Comment:
Referring to comment [1], we can reuse the Button component to prevent code duplication like the one here :)